src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,377 @@
+# Functional Near-Infrared Spectroscopy
+
+Support for functional Near-Infrared Spectroscopy (fNIRS) was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+## fNIRS recording data
+
+{{ MACROS___make_filename_template(datatypes=["nirs"], suffixes=["nirs", "events"]) }}
+
+Only the Shared Near Infrared Spectroscopy Format ([SNIRF](https://github.com/fNIRS/snirf))
+file specification is supported in BIDS. The SNIRF
+specification supports one or more fNIRS datasets to be stored in a single
+`.snirf` file. However, to be BIDS compatible, each SNIRF file MUST contain
+only a single run. A limited set of fields from the SNIRF specification are
+replicated  in the BIDS specification. This redundancy allows the data to be
+easily parsed by humans and machines that do not have a SNIRF reader at hand,
+which improves findability and tooling development.
+
+Original unprocessed fNIRS data in the native format, if different from SNIRF, can also
```

Comment:
```suggestion
Raw fNIRS data in the native format, if different from SNIRF, can also
```
By original unprocessed do you mean "raw"? or does it have additional meaning? Otherwise, maybe raw is the more used way to express it throughout the specs

Reply:
Raw often refers to the amplifier output data in fNIRS. As opposed to optical density or haemoglobin data. So we will leave it as Original unprocessed.